workers/unity/Packages/com.improbable.gdk.core/Tests/Editmode/GameObjectRepresentation/Readers/NonBlittableReaderTests.cs

```diff
@@ -0,0 +1,106 @@
+using System;
+using System.Collections.Generic;
+using Generated.Improbable.Gdk.Tests.BlittableTypes;
+using Generated.Improbable.Gdk.Tests.NonblittableTypes;
+using Improbable.Gdk.Core;
+using Improbable.Gdk.Core.EditmodeTests;
+using Improbable.Gdk.Core.GameObjectRepresentation;
+using Improbable.Gdk.Core.MonoBehaviours;
+using JetBrains.Annotations;
+using NUnit.Framework;
+using Unity.Entities;
+using UnityEngine;
+
+namespace Improbable.Gdk.Core.EditmodeTests.MonoBehaviours.Readers
+{
+    [TestFixture]
+    public class NonBlittableReaderTests
```

Comment:
Why is this a separate thing while the Blittable reader is in the ReaderWriterTestsBase?

Reply:
I see NB types as exceptional but they should still be proven to work as expected. The main difference is the `.Data` implementation, so that should still be tested separately from the main cases.